tests/cross/src/test/scala/tests/pc/CompletionIssueSuite.scala

```diff
@@ -155,6 +155,22 @@ object CompletionIssueSuite extends BaseCompletionSuite {
     topLines = Some(2)
   )
 
+  check(
+    "issue-813-multi",
+    """|package a
+       |
+       |object Main {
+       |  Array(1, 1,10)
+       |  .map(_ + 1) /* comment breaks completions */
```

Comment:
Are completions supposed to be working when the comment is truly multiline?

Reply:
I think they should. The code compiles, so we should provide completions. Though, it would be weird to write a code like that for sure :O

Reply:
I guess you're allowed to break the cascade of code calls with multiline comments and continue with a `.nextMethod(...)` on a new line, after all the comments should be stripped by the compiler sooner or later...
It's a weird practice tho, breaking the flow of calls like that, I wouldn't encourage it...